### ISSUE TITLE
workbench.userDataSync.actions.showConflicts error in dev tools

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode#202595]
```
log.ts:437   ERR Unable to create workbench contribution oe. Error: Cannot register two commands with the same id: workbench.userDataSync.actions.showConflicts
    at a (actions.ts:584:9)
    at oe.sb (userDataSync.ts:844:42)
    at oe.lb (userDataSync.ts:704:8)
    at new oe (userDataSync.ts:145:9)
    at b.j (instantiationService.ts:119:18)
    at b.createInstance (instantiationService.ts:85:18)
    at k (contributions.ts:187:25)
    at a (contributions.ts:163:10)
```

**Comment by alexr00** [comment]
Related: https://github.com/microsoft/vscode/pull/202318, https://github.com/microsoft/vscode/issues/202563

**Comment by sandy081** [comment]
Fixed by 7f825208bf67fe5f81131e0ed36dcb9fa063e403
